Gemfile.lock

```diff
@@ -115,6 +115,7 @@ PATH
       decidim-comments (= 0.9.0.pre)
       decidim-core (= 0.9.0.pre)
       kaminari (~> 1.0.1)
+      ransack
```

Comment:
ransack looks nice, maybe we should drop searchlight in favor of it? @deivid-rodriguez @josepjaume @mrcasals

Reply:
Looks good to me, although I just gave it a superficial look. Should we open an issue to discuss this?